tests/libres_tests/res/enkf/test_enkf_fs_manager2.py

```diff
@@ -19,7 +19,7 @@ def test_rotate(self):
         with ErtTestContext(self.config_file) as testContext:
             ert = testContext.getErt()
             fsm = ert.getEnkfFsManager()
-            self.assertEqual(0, fsm.getFileSystemCount())
+            self.assertEqual(1, fsm.getFileSystemCount())
```

Comment:
Is it ok to change this behavior? I resisted changing this as I added the magic strings to the getCurrentFileSystem() and switchFileSystem instead of the init.

Reply:
This is a consequence of opening the file system in python instead of C, previously the alloc in C added handled opening the first fs, then everything else was handled in python. It was then added to the `fs_manager` the first time it was called from python. Meaning that if I added a `switchFileSystem` to the same file system in this test before these changes the count would be 1. This change means the behavior is more consistent.